validator/db/manage_test.go

```diff
@@ -54,6 +58,220 @@ func TestMerge(t *testing.T) {
 	assertMergedStore(t, mergedStore, firstStorePubKeys, secondStorePubKeys, history)
 }
 
+func TestSplit(t *testing.T) {
```

Comment:
I redesigned the tests, I made helper functions less granular so that they can be reused by tests for merging and splitting. I hope most of these changes will make it easier to write tests for future functionalities, e.g. importing.